### PR TITLE
Alex   centered all games text 

### PR DESCRIPTION
**Pull request by alexhertel87**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules/
 **/**/.DS_Store
 .env
+package-lock.json
```
Comment (whuang3015) [inline .gitignore]: Package Lock needs to be in the file or else it won't load the necessary node packages

File: views/games.pug [hunk 2]
```diff
@@ -4,10 +4,11 @@ append head
...
     script(src="/javascripts/index.js" type="module" defer)
 block content
-    div.allGameBox
-        h1.allGamesHeader All Games
-        div.games
-            each game in games
-                div.game
-                    a.games(href=`/games/${game.id}/reviews`)
-                        img(src=`${game.imageSrc}` class="gameThumbnail")
+    body
```
Comment (whuang3015) [inline views/games.pug:7]: I don't think this is needed, the body tag already exists in the layout.pug that is extended.

File: views/genres.pug [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+    link(rel="stylesheet" href="/stylesheets/allGensPubs.css")
     script(src="/javascripts/index.js" type="module" defer)
 block content
-    .genres-container
-        h1 All Genres
-        .genres-container__thumbnails
-            each genre in genres
-                div
-                    a(href=`/genres/${genre.id}`) #{genre.genreName}
+    body
```
Comment (whuang3015) [inline views/genres.pug:7]: I don't think this body tag is needed, the body tag already exists in the layout.pug that is extended.

**Review by whuang3015 — CHANGES_REQUESTED** [review]
Please see comments. there's a few things that may need editing before it's clean to merge.

File: views/genres.pug (hunk 3 above)
Comment (alexhertel87) [reply views/genres.pug:7]: the body tag allowed me to style the whole page in line with the rest of the aesthetic

File: views/games.pug (hunk 2 above)
Comment (alexhertel87) [reply views/games.pug:7]: Body tag allowed me to easily style the rest of the page in line with rest of site aesthetic

**Review by alexhertel87 — COMMENTED** [review]
Reviewd suggested edits

File: .gitignore (hunk 1 above)
Comment (alexhertel87) [reply .gitignore]: I think I deleted it already from the pull req. But if that didn't work, I think you can just approve everything from the pull request besides the .gitignore and then delete it from the que

**Review by whuang3015 — APPROVED** [review]

**Pull request merged**